unity/Assets/Scripts/DebugInputField.cs

```diff
@@ -895,12 +895,41 @@ public void Execute(string command)
                     
                 case "poap":
                     {
-                        ServerAction action = new ServerAction();
-                        action.action = "PlaceObjectAtPoint";
-                        action.position = GameObject.Find("TestPosition").transform.position;
-                        action.objectId = "Book|+00.15|+01.10|+00.62";
-                        //action.rotation = new Vector3(0, 90, 0);
-                        PhysicsController.ProcessControlCommand(action);
+                        Dictionary<string, object> action = new Dictionary<string, object>();
+                        action["action"] = "PlaceObjectAtPoint";
+
+                        BaseFPSAgentController agent = AManager.agents[0];
+
+                        GameObject itemInHand = AManager.agents[0].ItemInHand;
+                        if (itemInHand != null) {
+                            itemInHand.SetActive(false);
+                        }
+
+                        RaycastHit hit;
+                        Ray ray = agent.m_Camera.ViewportPointToRay(new Vector3(0.5f, 0.5f, 0.0f));
+                        bool raycastDidHit = Physics.Raycast(ray, out hit, 10f, (1 << 8) | (1 << 10));
```

Comment:
Even though this is just for debug purposes, this functionality seems nice. Currently PlaceObjectAtPoint requires you to pass in a position that you obtain somewhere else, but it might be worth considering using a screenspace raycast as an optional way to "target" the position you are trying to place an object at.

This may require more testing of course, but we should keep a note that this functionality might be something good to build in to the actual action as well.